src/client-side-encryption/mongocryptdManager.ts

```diff
@@ -32,27 +34,30 @@ export class MongocryptdManager {
         .filter(arg => typeof arg === 'string')
         .every(arg => arg.indexOf('--idleShutdownTimeoutSecs') < 0)
     ) {
-      this.spawnArgs.push('--idleShutdownTimeoutSecs', 60);
+      this.spawnArgs.push('--idleShutdownTimeoutSecs', '60');
     }
   }
 
   /**
-   * @ignore
    * Will check to see if a mongocryptd is up. If it is not up, it will attempt
    * to spawn a mongocryptd in a detached process, and then wait for it to be up.
-   * @param {Function} callback Invoked when we think a mongocryptd is up
    */
-  spawn(callback) {
+  spawn(callback: Callback<void>) {
     const cmdName = this.spawnPath || 'mongocryptd';
 
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { spawn } = require('child_process') as typeof import('child_process');
+
     // Spawned with stdio: ignore and detatched:true
     // to ensure child can outlive parent.
     this._child = spawn(cmdName, this.spawnArgs, {
       stdio: 'ignore',
       detached: true
     });
 
-    this._child.on('error', () => {});
+    this._child.on('error', () => {
+      // perhaps questionable, but we swallow mongocryptd spawn errors.
+    });
```

Comment:
It is not exactly questionable, the source of "truth" for a successful launch is a mongocryptd process we can connect to. Since this channel of getting an error can occur at anytime we defer to the connection to mongocryptd being broken instead which is kind of our way of "monitoring" that it is still up.

Reply:
Right - but we could race `spawn` and `error` together, and once successfully spawned stop listening for errors. 

It's strange that there's no way to know if auto spawn was successful other than a failing auto encrypted command (which manifests itself as a failure to connect the mongocryptd client).

Reply:
This might be something that is captured in the specification (or should be). We can discuss the finer points in tech chat but for now let's leave something more descriptive than "perhaps questionable". I'm thinking let's capture the intention regardless of our takes on the approach. 

Something like:
> Add an empty error listener to squash child process errors
> If the process encountered an error we will learn of it when we fail to connect